dictionary/lib/dictionary/supervisor.ex

```diff
@@ -0,0 +1,11 @@
+defmodule Dictionary.Supervisor do
```

Comment:
# -2

This seems to be an artifact from an experiment—it doesn't seem to be used.